variables.tf

```diff
@@ -56,10 +56,17 @@ variable "transit_gateway_id" {
 variable "ram_resource_association_id" {
   description = "Identifier of the Resource Access Manager Resource Association"
```

Comment:
Can you clarify what that means?

Reply:
Sure. Let me know if this does the trick or not: https://aws.amazon.com/blogs/aws/new-aws-resource-access-manager-cross-account-resource-sharing/

TL;DR attempt: you have a shared resource and then you associate this shared resource with something.